README.md

```diff
@@ -48,6 +50,7 @@ The elements shown in the architecture diagram above are deployed by the [Cyral
     and with the database you plan to protect with this sidecar.
     * Set parameters `AssociatePublicIpAddress=true` and `LoadBalancerScheme='internet-facing'`
     to deploy a public sidecar.
+    * (Optional) Configure the Sidecar certificates in accordance with your deployment decision.
```

Comment:
Doesn't this make sense to leave it as is since one of the deployment decisions is to just use the self signed certificate generated by the cert manager.

Also, during POCs a customer may choose to load a certificate as well.

Reply:
But this is for a quickstart option and the idea is that the customer would be able to run it without having to go through any docs advanced options besides the steps in the quickstart. I would agree with that if you were referring to the production option. Also, it makes no sense to have these two sentences here referring to the certificate and not having them in the production sidecar.